packages/terra-consumer-nav/src/components/nav-help/NavHelp.jsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import Arrange from 'terra-arrange';
+import Grid from 'terra-grid';
+import Popup from 'terra-popup';
+import IconClose from 'terra-icon/lib/icon/IconClose';
+import NavHelpContent from './NavHelpContent';
+import IconInfo from '../../icons/IconInfo';
+import styles from './NavHelp.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  help: PropTypes.arrayOf(PropTypes.shape({
+    slug: PropTypes.string,
+    nav_type: PropTypes.oneOf(['GROUPING', 'EXTERNAL', 'APPLICATION', 'MODAL']),
```

Comment:
Removed slug and nav_type props as they are not used for Nav and Layout components.